alpine/server/Dockerfile

```diff
@@ -40,6 +40,6 @@ RUN echo "Installing GoCD ${GO_VERSION}" && wget -qO- $DOWNLOAD_URL/$GO_VERSION/
   |jar xf /dev/stdin && \
   # Note this rename won't work if /go-server already exists... (it'll move the dir inside /go-server)
   mv ./go-server-* ./go-server && \
-  sed -e 's/DAEMON=Y/DAEMON=N/' /go-server/default.cruise-server > /config/default/go-server && \
+  sed -e 's/DAEMON=Y/DAEMON=N/' /go-server/go-server.default > /config/default/go-server && \
```

Comment:
This change will work as the  Go version is set to ARG GO_VERSION=17.1.0-4511 or whenever this got introduced in server code. I tried to run it on ARG GO_VERSION=16.6.0-3590 and it didn't work.